AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -9,6 +9,8 @@ linksht.com#$##originalLink { display: block!important; }
 reddit.com#$#body > .footer-parent { height: calc(100%)!important; padding:0px!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/22766
 classicfm.com#$#.radioplayer { height:calc(100%)!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/35135
+ool.ru#$#.row > div[style*="height: 3632px;"] { height:calc(100%)!important; }
```

Comment:
it is not required on my end. Please check again.

Reply:
Checked again without rule, see screenshot:

<img width="711" alt="ool ru" src="https://user-images.githubusercontent.com/34266271/60688544-272ce180-9eb6-11e9-895f-5ccdf1798e1e.png">

I see height is not always that high and also requires sub-element to be modified:
`ool.ru#$#.row div.sameblockheight[style^="height: "] { height:calc(100%)!important; }`

Reply:
@zloyden

Reply:
I had the same thing that's why I added two tweaks from BlazDT. With this rules the page looks ok.
Here is a screenshot without tweaks.
<details><summary>Screenshot:</summary>

![1](https://user-images.githubusercontent.com/33488229/60742406-f0b69b80-9f75-11e9-8068-98b59623d4ab.png)
</details><br/>